tests/bindings/javascript/importsource.test.js

```diff
@@ -13,3 +13,43 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+
```

Comment:
Have only one empty line rather than two?

Reply:
I think there is only one empty line?

Reply:
Lines 16 and 17 are empty, so 2 empty lines while we have 1 empty line elsewhere (see, for instance, `tests/bindings/javascript/variable.test.js` which you have "fixed" in this PR).

Reply:
When I look at the file now it only has one empty line but it is showing two empty lines here. I don't know how to get this diff to update to the actual file content.

Reply:
See here: https://github.com/hsorby/libcellml/blob/js-importsource/tests/bindings/javascript/importsource.test.js
In this view I only see one empty line.

Reply:
Hmm, indeed... Must be a GitHub issue. Ok, resolving this conversation then.